app/routes/task_routes.py

```diff
@@ -0,0 +1,124 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+from datetime import datetime
+import requests
+from dotenv import load_dotenv
+import os
+
+
+load_dotenv()
```

Comment:
`load_dotenv()` only needs to be called on startup to load the values in .env 

After it's executed from create_app() we only need to import os and get the key from os in task_routes.py